eth/abc.py

```diff
@@ -1135,27 +1213,6 @@ def persist_uncles(self, uncles: Tuple[BlockHeaderAPI]) -> Hash32:
     #
     # Transaction API
     #
-    @abstractmethod
```

Comment:
These were unused everywhere in the code base so I removed them from the abstract class and the implementation class